packages/devtools/lib/src/timeline/frame_flame_chart.dart

```diff
@@ -84,39 +123,31 @@ class FrameFlameChart extends CoreElement {
     }
 
     if (frame != null) {
-      _render(frame);
+      _render();
     }
   }
 
-  void _render(TimelineFrame frame) {
+  void _render({bool reRender = false}) {
```

Comment:
instead of reRender,
consider abstracting the UI so there is a separate method called for the initial UI initialization and just updating the zoom. Seems like more logic is called than needed when just updating the zoom. 
If that is hard maybe go with a different parameter name than reRender. Maybe 
bool reuseElements
or
bool buildElements.

Reply:
done.